KsApi/mutations/inputs/SignInWithAppleInput.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public struct SignInWithAppleInput: GraphMutationInput {
```

Comment:
If we conform `SignInWithAppleInput` to `Encodable` then we don't need the `toInputDictionary()` function below.

Reply:
Done! Also noticed we were missing tests for this, so I added those too.